Dockerfile.OCP

```diff
@@ -0,0 +1,46 @@
+FROM alpine:3.16
+
+#    REGISTRY=https://registry.npmjs.org \
+ARG ENT_VERSION
+ARG ENT_CLI_VERSION
+ARG KUBECTL_RELEASE
+
+ENV BASH_ENV=/usr/local/bin/scl_enable \
+    ENT_VERSION=$ENT_VERSION \
+    PATH=$PATH:/home/ent/.entando/ent/$ENT_VERSION/cli/$ENT_VERSION/bin:/home/ent/.entando/ent/$ENT_VERSION/opt/node-v14.20.0/bin \
```

Comment:
could we use in PATH, ENTANDO_ENT_HOME the pattern 
`/home/ent/.entando/ent/$ENT_VERSION/cli/$ENT_CLI_VERSION` as in Dockerfile.Github?
or is there a different behavior here?

Reply:
@antromeo you are right. Good catch.